### PR TITLE
[ZH] Add `People\The_Team\Beatmap_Nominators\Becoming_a_Beatmap_Nominator`

### PR DESCRIPTION
**Pull request by YumeMuzi**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+以下这些是对提交摸的特殊要求。任何一条不满足都可能导致 BN 申请被拒。
+
+- **覆盖所有难度等级。**
+- **识别出摸图工具无法显示出的，不符合上架标准 (unrankable) 的问题。如不准确的歌曲时间轴或是歌曲元数据问题。**
+- **比较谱面不同段落，并提出问题或改进建议。**
+- **评述谱面的整体元素。如节奏、物件间距、物件运动、物件密度、不同段落的差异、谱面一致性。**
+- **清晰区分单一出现的问题摸和广泛的全谱面问题摸。**
+- **提出问题或改进建议时，总是从谱师角度出发考虑。**
+
+### 如何挑选潜在的 BN
```
Comment (RushFTK) [inline wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md:30]:
```suggestion
### 潜在的 BN 是如何被挑选的
```
这个意见是可选的

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+提交的摸应该展示了申请者有能力分析并提升谱面质量，达到了担任 BN 的水平要求。
+
+以下这些是对提交摸的特殊要求。任何一条不满足都可能导致 BN 申请被拒。
+
+- **覆盖所有难度等级。**
+- **识别出摸图工具无法显示出的，不符合上架标准 (unrankable) 的问题。如不准确的歌曲时间轴或是歌曲元数据问题。**
+- **比较谱面不同段落，并提出问题或改进建议。**
+- **评述谱面的整体元素。如节奏、物件间距、物件运动、物件密度、不同段落的差异、谱面一致性。**
+- **清晰区分单一出现的问题摸和广泛的全谱面问题摸。**
+- **提出问题或改进建议时，总是从谱师角度出发考虑。**
```
Comment (RushFTK) [inline wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md:28]:
```suggestion
- **提出问题或改进建议时，考虑谱面作者的意图。**
```
这个意见是可选的。从谱师角度出发还有可能是把自身假想成自己做这个歌曲时的想法，而这里其实更强调是假想成对方，所以我更倾向于直译。

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+### 如何计算最低分数
+
+- 这取决于申请者过去几个月所取得的总`摸图分数 (ModScore)`。这个值由以下公式计算得出：
+
+![摸图分数 (ModScore) 公式](/wiki/shared/Modscore_new_wiki.png "摸图分数 (ModScore) 公式")
+
+`m` = 单月内摸图的最低限制
+`M` = 单月内摸图的总数
+
+- [网址](http://bn.mappersguild.com/bnapps)的主页面含有能够计算`摸图分数 (ModScore)` 的工具。
```
Comment (RushFTK) [inline wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md:57]:
```suggestion
- [该网页](http://bn.mappersguild.com/bnapps)含有能够计算`摸图分数 (ModScore)` 的工具。
```

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+  - osu!：`m` = 4，且获得的 kudosu 数量大于等于 200。
+- 建议尽量满足或者超出这些预期要求，因为它们可能会因为申请数量不同而略微变动。
+
+## 冷却期
+
+取决于你所在的状态不同，申请失败再次申请之前所需的冷却期和摸图分数 (ModScore) 要求也会不同。
+
+### 申请失败
+
+当申请 BN 失败后，申请者将需要满足以下要求，以再次提交申请。
```
Comment (RushFTK) [inline wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md:70]:
```suggestion
申请者在申请 BN 失败后，再次提交申请前需满足以下要求：
```

**Review by RushFTK — DISMISSED** [review]
all changes are optional. after conversation resolved, pages can be merged.

**Comment by Walavouchey** [comment]
@RushFTK should the changes here be applied?

**Comment by RushFTK** [comment]
@Walavouchey wait him for confirmed

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md (hunk 1 above)
Comment (YumeMuzi) [reply wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md:30]: 第二个翻译腔，保持原来的
translationese.

File: wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md (hunk 2 above)
Comment (YumeMuzi) [reply wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/zh.md:28]: 不矛盾啊，必须站在对方角度想问题，包括为什么这里要放物件，如果我是谱师，我会这么放吗，为什么等等，所以保留
Putting yourself in their shoes is the basis of modding. keep

**Review by RushFTK — APPROVED** [review]

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**